<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleDialog</class>
 <widget class="QDialog" name="SaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление продажи</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="Group4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Сумма продажи</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QPushButton" name="btnCancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Продавец</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Дата продажи</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="cbCustomer"/>
    </item>
    <item row="0" column="2">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Покупатель</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="btnOk">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="txtAmount"/>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="cbVehicle"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Транспортное средство</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="cmbDealer"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
